pkg/features/kube_features.go

```diff
@@ -828,6 +834,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	GracefulNodeShutdown:                           {Default: false, PreRelease: featuregate.Alpha},
 	ServiceLBNodePortControl:                       {Default: false, PreRelease: featuregate.Alpha},
 	MixedProtocolLBService:                         {Default: false, PreRelease: featuregate.Alpha},
+	NamespaceDefaultLabelName:                      {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
We need to remove feature gating, it looks like, @thockin and @liggitt  hugged it out and are on the same page - stomp stomp stomp !!!